.gitignore

```diff
@@ -103,4 +103,7 @@ vertexai-sdk-test-data
 mocks-lookup.ts
 
 # temp changeset output
-changeset-temp.json
\ No newline at end of file
+changeset-temp.json
+
+# Vertex AI integration test Firebase project config
+packages/vertexai/integration/firebase-config.ts
```

Comment:
Maybe just include this file with an empty object exported so that the IDE doesn't get mad, and if you run it locally, you overwrite it, and if you run it in CI, the workflow overwrites it. Maybe we should do this in e2e/ too.

Reply:
That's a good idea, done

Reply:
I did this, but it turns out that if it's tracked in git, changes will be tracked even if it's in the .gitignore. this means if someone updates this with the testing config, they may accidentally commit the change and upload it to git.

I believe we can't track this file with git while ignoring changes. I think we should just remove the file for now.